Formula/libjwt.rb

```diff
@@ -0,0 +1,37 @@
+class Libjwt < Formula
+  desc "JSON Web Token C library"
+  homepage "https://github.com/benmcollins/libjwt"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "check" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jansson"
+  depends_on "openssl"
+
+  def install
+    system "autoreconf", "-v", "-i"
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "all"
+    system "make", "check"
```

Comment:
Is this security software?

Reply:
What do you mean? This `make check` runs the `libjwt` library test suite. The library itself is just a C library to implement JSON Web Tokens and operate with them.